### ISSUE TITLE
Libsodium error (Linking failure in asm.js: Unexpected stdlib member) when using inside Jest

### ISSUE THREAD

**Issue by pixelastic** [TankerHQ/sdk-js#19]
### What is the current behavior?

When importing `@tanker/client-node` and creating a `new Tanker()` inside a Jest test, the following warning is displayed:

```
(node:2735814) V8: /home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium/dist/modules/libsodium.js:13 Linking failure in asm.js: Unexpected stdlib member
(Use `node --trace-warnings ...` to show where the warning was created)
```

The same code works fine when used outside of Jest.

### What is the expected behavior?

I would expect no warning to be displayed (or a way to prevent the warning).

### Can you provide a list of simple steps that reproduce the problem?

I created a repository reproducing the issue here: https://github.com/pixelastic/issue-tanker-jest-libsodium

1. Clone the repo
2. `yarn install`
3. Run the test with `yarn run test`
4. See the warning

![image](https://user-images.githubusercontent.com/283419/121366555-de032900-c939-11eb-9706-1f78e85a4faf.png)

You can try running `yarn run run` which runs the same code, but outside of Jest and see that no warning is displayed.

### Do you have any logs you can provide? (from the browser's console for instance)

If run I run the method with `--trace-warnings` I have the following stacktrace:

```
(node:2750494) V8: /home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium/dist/modules/libsodium.js:13 Linking failure in asm.js: Unexpected stdlib member
    at /home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium/dist/modules/libsodium.js:23:1
    at /home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium/dist/modules/libsodium.js:6:9
    at Object.<anonymous> (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium/dist/modules/libsodium.js:10:3)
    at Runtime._execModule (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/jest-runtime/build/index.js:1394:24)
    at Runtime._loadModule (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/jest-runtime/build/index.js:996:12)
    at Runtime.requireModule (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/jest-runtime/build/index.js:828:12)
    at Runtime.requireModuleOrMock (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/jest-runtime/build/index.js:1017:21)
    at /home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium-wrappers/dist/modules/libsodium-wrappers.js:8:26
    at Object.<anonymous> (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/libsodium-wrappers/dist/modules/libsodium-wrappers.js:16:2)
    at Runtime._execModule (/home/tim/local/www/forks/issue-tanker-jest-libsodium/node_modules/jest-runtime/build/index.js:1394:24)
```

## Environment

Tanker SDK version: 2.14.0
Web browser: N/A
Web browser version: N/A
OS: Ubuntu 20.04.2 LTS 64bits

## What I tried already

I found similar issues online (with the same libsodium error) in other crypto libraries and it seems to come from Jest that is overwriting some native Node constants (like Buffer or Uint8Array). 

I checked in `./node_modules/libsodium/dist/modules/libsodium.js` and saw that it's referencing those types:
![image](https://user-images.githubusercontent.com/283419/121367622-beb8cb80-c93a-11eb-97eb-36c2b3e7ba3c.png)

I did some quick tests, checking the output of `UintArray.toString()` and the other native types right before calling `new Tanker()` and they all outputted `[ native code]` which made me believe they were not changed by Jest. I also checked Buffer and ArrayBuffer and they also seem to map to their native code, so I don't know what Jest changed that would make this fail.

The warning is not blocking *per se*, but I would love to get rid of it. Best case scenario it's just noise, worst case scenario I have a different behavior in my tests and in my app. In both cases, I'd like to make it disappear.

**Comment by JMounier** [comment]
Hi @pixelastic, thanks for the feedback.

We are upgrading our libsodium dependency in the next release and it removes the warning. Have a look at https://github.com/TankerHQ/issue-tanker-jest-libsodium. The two difference with your test are:
- Use an alpha version of Tanker/client-node: `2.15-alpha1`
- Call `new Tanker(...).start()` to ensure the sodium code is present.

Can you confirm that version `2.15-alpha1` fixes the warning on your end too?

**Comment by JMounier** [comment]
We just released version `2.15.0`.

**Comment by pixelastic** [comment]
Thanks a lot, I'll give it a try and let you know.
